src/app/components/autocomplete/autocomplete.component.html

```diff
@@ -0,0 +1,25 @@
+<form>
```

Comment:
Can we leave the form in AppComponent, and the submit button?

Reply:
fixed in the new commit